static/sass/patterns/_forms.scss

```diff
@@ -25,25 +25,3 @@
 .form-actions {
   text-align: right;
 }
-
-.btn {
-  &,
-  #{$link} {
-    @include contrasted('action');
-    background-image: none; // remove link underline when styling as button...
-    border-color: transparent;
-    border-style: solid;
-    border-width: 1px;
-    display: inline-block;
-    padding: 0.5rem 1rem;
-    text-decoration: none;
-    text-shadow: none;
-    transition: color 150ms, background-color 150ms;
```

Comment:
Should some of the rest of these styles also get moved over to the new `_buttons.scss`?